services/payments/src/simcore_service_payments/services/notifier_email.py

```diff
@@ -153,6 +176,17 @@ async def _create_user_email(
     if product.bcc_email:
         msg["Bcc"] = product.bcc_email
 
+    # Invoice attachment
+    pdf_response = await _get_invoice_pdf(payment.invoice_pdf)
+
+    pattern = re.compile(r'filename="(?P<filename>[^"]+)"')
```

Comment:
If you compile, then it is better to do it only once. Add this as a constant that is create upon import